src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -75,15 +75,18 @@ typedef struct SpecialRespawnInfo {
 } SpecialRespawnInfo; // size = 0x10
 
 typedef enum AnimSfxType {
-    /* 1 */ ANIMSFX_TYPE_1 = 1,
-    /* 2 */ ANIMSFX_TYPE_2,
-    /* 3 */ ANIMSFX_TYPE_3,
-    /* 4 */ ANIMSFX_TYPE_4,
-    /* 5 */ ANIMSFX_TYPE_5,
-    /* 6 */ ANIMSFX_TYPE_6,
-    /* 7 */ ANIMSFX_TYPE_7,
-    /* 8 */ ANIMSFX_TYPE_8,
-    /* 9 */ ANIMSFX_TYPE_9
+    /* 1 */ ANIMSFX_TYPE_NO_OFFSET = 1,
+    /* 2 */ ANIMSFX_TYPE_WITH_FLOOR_OFFSET,
+    /* 3 */ ANIMSFX_TYPE_WITH_FLOOR_AND_AGE_OFFSETS,
+    /* 4 */ ANIMSFX_TYPE_WITH_AGE_OFFSET,
```

Comment:
Even though its true that internally the system applies offests to sfx ids to create other ids, I dont think the type names necessarily need to mention that. 

While typing out my initial idea suggestions, they turned out to be identical to what MM already has:
```c
    /*  1 */ ANIMSFX_TYPE_GENERAL = 1,
    /*  2 */ ANIMSFX_TYPE_FLOOR,
    /*  3 */ ANIMSFX_TYPE_FLOOR_BY_AGE,
    /*  4 */ ANIMSFX_TYPE_VOICE,
```
MMs current `ANIMSFX_TYPE_GENERAL` I can see being `DEAFULT` or `NORMAL` as well, I dont have too much of a preference.

 `ANIMSFX_TYPE_VOICE` does adjust by age, but *also* adjusts pitch for dark link. So this is why just mentioning "age" is not fully sufficient. Hopefully specifying voice is enough to imply that it will adjust. This fact can be added in a comment if you feel its necessary.

Reply:
Done